_posts/2015-06-26-ux-part-3.md

```diff
@@ -0,0 +1,122 @@
+
+##Third Round
+
+... back to the drawing board.
+
+<img src="/images/state-9.png" alt="Mock-ups of multiple rendering settings per user per image." style="width: 600px;"/>
+
+*Multiple rendering settings per user.*
+
+Yes, but no. They liked that, but wanted to be able to give each setting a unique name.
+
+So we needed to add editing buttons and dialogue boxes.
+
+<img src="/images/state-10.png" alt="Mock-ups of multiple rendering settings per user per image with name and editing dialogs." style="width: 600px;"/>
+
+*Let them have names!*
+
+And this will hopefully be implemented in the next round of UI improvements.
+
+Another important advantage with this workflow is that with all the scoping done well in advance of actual implementation, it helps to avoid conflict with other changes in the pipeline. So now if another change is muted that affects the Preview pane, Full Viewer or rendering settings, it is relatively easy to make sure that it fits in with these changes.
+
+##Release
+
+The UX team’s job doesn’t end with code implementation, as we don’t just abandon the user at this stage. With the OME’s continuous integration development system, the code, the developer documentation and the User Help are all updated and held in repositories ready for release.
+
+So when all the testing is complete and everyone’s happy, Sébastien presses The Big Red Button and the version release of all the elements is done at once. 
+
+<img src="/images/release.png" alt="Image showing the release download page, developer documentation and User Help." style="width: 600px;"/>
+
+*Similtaneous release of all elements.*
+
+The benefits to the team of this coordination using the continuous integration and release system have been huge, making the testing and review of updates as painless as humanly possible. It is a vast improvement on what happened before.
+
+##Iterations Count
+
+This is the Scoping folder on my Mac - 32 separate scoping exercise so far. Some, like  number 4 - Tagging, have up to nine versions of iteration.
+
+<img src="/images/iterations.png" alt="Image showing the number of files in a scoping folder." style="width: 600px;"/>
+
+*The extent of the scoping work and iterations.*
+
+Once we have implemented something we do try and remember to record screenshots of the released version and add them to the team Google Drive as a record of the final implementation.
+
+##In Summary
+
+So while I admit user-centred design is not always the easiest road to travel in small, expert system developments, it still gives good results. As we have pressed on with this approach, we have seen what were a pretty high level of user complaints, and sometimes even downright hostility, slowly morph into much more positive feedback. From: “why can’t I do this, your software is useless” to: “it would be really cool if I could also do that”.
```

Comment:
I'm not sure if Jason will be happy with being this negative about our earlier incarnations, could we just say, feedback is much more positive (although I think it's fine to leave in the example comment, just not the "pretty high level od user complaints, and sometimes even downright hostility" bit) ?